README.md

```diff
@@ -2,10 +2,12 @@
 
 The repository walks you through best practices, culture, and technological corner cases on PU Library. These are the parts that are tricky to fully automate or in some cases not possible to. These are also the shared norms and best practices agreed upon across teams in PUL IT which, because they are shared, do not fit solely into one team's handbook.
 
-Each developer team at PU Library also has its own team handbook:
+Each team at PU Library also has its own team handbook:
```

Comment:
Suggestion: "The developer and operations teams also have their own team handbooks" since PU Library is quite a broad term :)

Reply:
Addressed in 51ad4112904e5b6ada96726a11d6241089453f35